packages/migrations/src/migrate.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import { Deployer } from '@0xproject/deployer';
+import { devConstants } from '@0xproject/dev-utils';
+import { logUtils } from '@0xproject/utils';
+import * as path from 'path';
+
+import { runMigrationsAsync } from './migration';
+
+(async () => {
+    const deployerOpts = {
+        jsonrpcUrl: 'http://localhost:8545',
+        artifactsDir: path.resolve('src', 'artifacts'),
+        networkId: 50,
```

Comment:
This appears to be a testrpc only migrate. When we deploy to kovan/mainnet will we create a seperate migrate file or refactor to provide this with parameters?

Reply:
Yes correct. Other migrations will require this to be further refactored.